### PR TITLE
Add movie remove speaker

### PR DESCRIPTION
**Pull request by pwalczysko**
Adding Kobayashi video, removing PW from speakers list.

cc @will-moore @sbesson

### CODE REVIEW

File: events/ome-community-meeting-2020/day2/index.html [hunk 1]
```diff
@@ -10,7 +10,8 @@
       vid_title1: Ontology-based imaging data integration using OME-OWL
       vid_speaker1: "Norio Kobayashi"
       vid_aff1: RIKEN
-      vid_link1: "../../../img/icons/placeholder_video.svg"
+      vid_link1: "https://downloads.openmicroscopy.org/presentations/2020/Dundee/main-talks/Norio_Kobayashi.mp4"
+      vid_poster3: "/img/video/norio_kobayashi.png"
```
Comment (sbesson) [inline events/ome-community-meeting-2020/day2/index.html:14]: Is this file supposed to be added as part of this PR?

Comment (pwalczysko) [reply events/ome-community-meeting-2020/day2/index.html:14]: Yes, of course, thank you @sbesson

Comment (sbesson) [inline events/ome-community-meeting-2020/day2/index.html:14]:
```suggestion
      vid_poster1: "/img/video/norio_kobayashi.png"
```

**Comment by will-moore** [comment]
Looks good. 👍

**Comment by pwalczysko** [comment]
Thanks @sbesson @will-moore

**Pull request merged**